.github/workflows/ci.yml

```diff
@@ -203,20 +203,6 @@ jobs:
           $MAVEN ${MAVEN_TEST} -T 1C clean verify -DskipTests -Dair.check.skip-all=true ${MAVEN_GIB} -Dgib.buildUpstream=never -P errorprone-compiler \
             -pl '!:trino-docs,!:trino-server,!:trino-server-rpm'
 
-  web-ui-checks:
```

Comment:
maybe we need to change this to instead verify if the build causes the lock files to change - which suggests something needs checking in.

Reply:
@electrum thoughts?